src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -93,7 +98,7 @@ public class Elevator extends SubsystemBase {
   private Consumer<Double> consumerMMJerk;
 
   public Elevator() {
-    // TODO: set up for canivore
+    // TODO: set up for canivore (Do we still need this? -Jackson)
```

Comment:
No we don't need it. It appears everything but the drivebase is expected to stay on the RIO bus.